### DISCUSSION TITLE
SCP do not load to QGIS 3.34.9-Prizren

### DISCUSSION THREAD

**Discussion by oguerrarea** [semiautomaticgit/SemiAutomaticClassificationPlugin#367] [General]
Your SCP is installed via the plugins menu in the QGIS. The programs is installed but do not load. The folowing message appears:

"Semi-Automatic Classification Plugin: Warning. Python library Remotior Sensus is outdated. This could cause errors, please update Remotior Sensus"

Is that true? If so, where is that Remotior Sensus to download and install it?

**Comment by semiautomaticgit** [comment]
Hello @oguerrarea,
please follow these installation steps to install the SCP and Remotior Sensus 
https://semiautomaticclassificationmanual.readthedocs.io/en/latest/installation.html

**Comment by oguerrarea** [comment]
Thanks a lot Luca.
O. Guerra
